serializers.py

```diff
@@ -58,7 +64,28 @@ def get_is_completed(self, instance):
 
             return instance.has_already_ended
 
+        def get_status_type(self, instance):
```

Comment:
I now feel that this should be a part of CMS as this is not related to FacApp. You can simply check for `isCompleted ` there.